pkg/noderesourcetopology/pluginhelpers.go

```diff
@@ -32,21 +32,14 @@ import (
 )
 
 func findNodeTopology(nodeName string, nodeResTopoPlugin *nodeResTopologyPlugin) *topologyv1alpha1.NodeResourceTopology {
-	klog.V(5).InfoS("Namespaces for nodeResTopoPlugin", "namespaces", nodeResTopoPlugin.namespaces)
-	for _, namespace := range nodeResTopoPlugin.namespaces {
-		klog.V(5).InfoS("Lister for nodeResTopoPlugin", "lister", nodeResTopoPlugin.lister)
-		// NodeTopology couldn't be placed in several namespaces simultaneously
-		lister := nodeResTopoPlugin.lister
-		nodeTopology, err := (*lister).NodeResourceTopologies(namespace).Get(nodeName)
-		if err != nil {
-			klog.V(5).ErrorS(err, "Cannot get NodeTopologies from NodeResourceTopologyNamespaceLister")
-			continue
-		}
-		if nodeTopology != nil {
-			return nodeTopology
-		}
+	klog.V(5).InfoS("Lister for nodeResTopoPlugin", "lister", nodeResTopoPlugin.lister)
+	// NodeTopology couldn't be placed in several namespaces simultaneously
+	lister := nodeResTopoPlugin.lister
+	nodeTopology, err := (*lister).Get(nodeName)
+	if err != nil {
+		klog.V(5).ErrorS(err, "Cannot get NodeTopologies from NodeResourceTopologyLister")
```

Comment:
Could you check if it's possible to return a non-nil `nodeTopology` along with a non-nil `err`? If so, we should explicitly `return nil` below this line.

Reply:
No there is no case where a non-nil nodeTopology is returned with a non-nil err. I have updated as per your recommendation.